### PR TITLE
fix: improvements and clarifications to community pool proposals

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

File: pages/dev/governance/community-pool-proposals.mdx [hunk 1]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
 3. Once the feedback window has passed, you can submit the Proposal for an on-chain governance vote.
 
 ## On-chain governance proposal
-Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)) and a local Axelar node.
```
Comment (sergeynog) [inline pages/dev/governance/community-pool-proposals.mdx:15]: via an Axelar node [which may be local or remote]

File: pages/dev/governance/community-pool-proposals.mdx [hunk 2]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
...
+Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)).
 
 ### Prerequisites
 
 * You must have [axelard](/node/config-node#download-binaries-and-initialize-configuration)
-* You must have a local Axelar node running
-* You must have an account named `validator` with at least **2000AXL** balance
+* You must have an account with at least **2,000AXL** balance
```
Comment (sergeynog) [inline pages/dev/governance/community-pool-proposals.mdx:20]: or find a community member that can fund the 2000 AXL deposit on your behalf within 1 hour of posting a proposal.

File: pages/dev/governance/community-pool-proposals.mdx [hunk 3]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
...
+* You must choose an [RPC endpoint](/resources), run an Axelar node
```
Comment (sergeynog) [inline pages/dev/governance/community-pool-proposals.mdx:21]: or run an axelar node.

File: pages/dev/governance/community-pool-proposals.mdx [hunk 4]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
...
 ```
 
-### Submit a Proposal
-Now that you have a wallet named `validator` with 2000AXL, you can submit a proposal on-chain.
+### Submit a proposal
+Now that you have a account named `wallet` with 2,000AXL, you can submit a proposal on-chain.
 
 ```bash
-$ axelard tx gov submit-proposal community-pool-spend <file> --from validator --gas auto --gas-adjustment 1.2
+$ axelard tx gov submit-proposal community-pool-spend <file> --from wallet --node <RPC Endpoint> --gas auto --gas-adjustment 1.2
```
Comment (sergeynog) [inline pages/dev/governance/community-pool-proposals.mdx:45]: link axelar RPC providers from the resource page.

**Review by sergeynog — CHANGES_REQUESTED** [review]

File: pages/dev/governance/community-pool-proposals.mdx [hunk 5]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
...
 
 # display key address 
 $ axelard keys list
 
-# send funds from an existing wallet to this validator account
-$ axelard tx bank send [account_with_fund_address] [validator_key_address] 2000000000uaxl
+# fund the wallet with 2,000 AXL
+
+# For example, if you have another local wallet, you could transfer the funds with:
+$ axelard tx bank send [account_with_fund_address] [wallet_address] 2000000000uaxl
```
Comment (milapsheth) [inline pages/dev/governance/community-pool-proposals.mdx:38]: They can send funds via Keplr wallet, or get it from a CEX. They can check the balance on axelarscan. But they can use this command to send the funds back

File: pages/dev/governance/community-pool-proposals.mdx [hunk 6]
```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
...
 
-### Create a validator account
+### Create a local wallet
+If you don't already have a local wallet, you can create one.
+
 ```bash
-# create an account validator
-$ axelard keys add validator 
+# create an account named `wallet`
+$ axelard keys add wallet
```
Comment (milapsheth) [inline pages/dev/governance/community-pool-proposals.mdx:29]: They should save the generated mnemonic

File: pages/dev/governance/community-pool-proposals.mdx [hunk 7]
```diff
@@ -51,8 +56,12 @@ $ axelard tx gov submit-proposal community-pool-spend <file> --from validator --
 		"recipient": "<fund_recipient_address>",
 		"deposit": "2000000000uaxl"
 }
+
 ```
 
+Once the proposal passes or fails (without `NoWithVeto`), the deposit will be refunded.
```
Comment (milapsheth) [inline pages/dev/governance/community-pool-proposals.mdx:62]: Add the command/docs on how someone else can provide 2000 AXL. Feel free to test it out on testnet/devnet

File: pages/dev/governance/community-pool-proposals.mdx [hunk 8]
```diff
@@ -1,8 +1,8 @@
-# Community Grants Proposals
+# Community Pool Proposals
```
Comment (sergeynog) [inline pages/dev/governance/community-pool-proposals.mdx:1]: this should not be under developers, as anyone can/should be able to submit these. 
maybe create its own top level page for governance? or put under resources?

File: pages/dev/governance/community-pool-proposals.mdx (hunk 4 above)
Comment (Olanetsoft) [reply pages/dev/governance/community-pool-proposals.mdx:45]: Updated

File: pages/dev/governance/community-pool-proposals.mdx (hunk 6 above)
Comment (Olanetsoft) [reply pages/dev/governance/community-pool-proposals.mdx:29]: Added

File: pages/dev/governance/community-pool-proposals.mdx (hunk 8 above)
Comment (Olanetsoft) [reply pages/dev/governance/community-pool-proposals.mdx]: Added to the resources category.

File: pages/dev/governance/community-pool-proposals.mdx (hunk 7 above)
Comment (Olanetsoft) [reply pages/dev/governance/community-pool-proposals.mdx]: Could you help me with the full command so that I can update it as requested?

**Review by sergeynog — APPROVED** [review]

**Pull request merged**